pkg/daemon/metrics.go

```diff
@@ -55,11 +55,11 @@ var (
 			Help: "state of kubelet health monitor",
 		})
 
-	// MCDRebootErr logs success/failure of reboot and errors
+	// MCDRebootErr logs failure of reboot and tallys count
 	MCDRebootErr = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Name: "mcd_reboot_err",
-		}, []string{"node", "message", "err"})
+		}, []string{"node", "err"})
```

Comment:
Is it feasible to the remove the `err` label altogether? In the normal error case the label `failed to run reboot` doesn't seem to add any information.
The other usage should never be reached anyway, so I'd argue to probably remove that.

Reply:
Good point, I'm going to spend some time rethinking this. Thank you!